infrastructure/dogfood/terraform/aws/ecs.tf

```diff
@@ -167,23 +167,23 @@ resource "aws_ecs_task_definition" "backend" {
             valueFrom = aws_secretsmanager_secret.database_password_secret.arn
           },
           {
-            name      = "FLEET_MDM_APPLE_SCEP_CA_CERT_PEM"
+            name      = "FLEET_MDM_APPLE_SCEP_CERT_BYTES"
             valueFrom = "${aws_secretsmanager_secret.scep.arn}:crt::"
           },
           {
-            name      = "FLEET_MDM_APPLE_SCEP_CA_KEY_PEM"
+            name      = "FLEET_MDM_APPLE_SCEP_KEY_BYTES"
             valueFrom = "${aws_secretsmanager_secret.scep.arn}:key::"
           },
           {
             name      = "FLEET_MDM_APPLE_SCEP_CHALLENGE"
             valueFrom = "${aws_secretsmanager_secret.scep.arn}:challenge::"
           },
           {
-            name      = "FLEET_MDM_APPLE_MDM_PUSH_CERT_PEM"
+            name      = "FLEET_MDM_APPLE_APNS_CERT_BYTES"
             valueFrom = "${aws_secretsmanager_secret.apn.arn}:FLEET_MDM_APPLE_MDM_PUSH_CERT_PEM::"
```

Comment:
note that here and below I'm not updating the value of the secrets manager key, assuming that we're okay with that at least for now.